pybalance/utils/balance_calculators.py

```diff
@@ -248,39 +248,56 @@ def _to_list(self, candidate_populations):
 
     @reshape_output
     def distance(
-        self, candidate_populations: Union[pd.DataFrame, torch.Tensor, np.ndarray]
+        self,
```

Comment:
seemingly inconsistent use of 'pool' to 'full population data' name change seen in _fetch_features?